src/components/claims/claim-details/components/ClaimInformation.tsx

```diff
@@ -93,96 +100,141 @@ const ClaimInformation: React.SFC<Props> = ({
+                    onChange={async (event) => {
+                      await updateClaimState({
+                        variables: {
+                          id: claimId,
+                          state: validateSelectOption(event),
+                        },
+                      })
+                      await refetchPage()
+                    }}
+                  >
+                    {Object.values(ClaimState).map((s) => (
+                      <MuiMenuItem key={s} value={s}>
+                        {s}
+                      </MuiMenuItem>
+                    ))}
+                  </MuiSelect>
+                </StatusWrapper>
+                <EmployeeClaimWrapper>
+                  <MuiInputLabel shrink>Employee Claim</MuiInputLabel>
+                  <MuiSelect
+                    value={coveringEmployee ? 'True' : 'False'}
+                    onChange={async (event) => {
+                      await setCoveringEmployee({
+                        variables: {
+                          id: claimId,
+                          coveringEmployee: validateSelectEmployeeClaimOption(
+                            event,
+                          ),
+                        },
+                      })
+                      await sleep(1000)
+                      await refetchPage()
+                    }}
+                  >
+                    <MuiMenuItem key={'True'} value={'True'}>
+                      True
+                    </MuiMenuItem>
+                    <MuiMenuItem key={'False'} value={'False'}>
+                      False
+                    </MuiMenuItem>
+                  </MuiSelect>
+                </EmployeeClaimWrapper>
+                <EmployeeClaimWrapper>
+                  <MuiInputLabel shrink>Add Contract Id to Claim</MuiInputLabel>
+                  <MuiSelect
+                    value={
+                      claim && claim.contract
+                        ? claim.contract.contractTypeName
+                        : 'Select contract Id'
+                    }
+                    onChange={async (event) => {
+                      await setAddContractIdToClaim(
+                        addContractIdToClaimOptions({
+                          claimId,
+                          memberId,
+                          contractId: event.target.value,
+                        }),
+                      )
+                      await sleep(1000)
+                      await refetchPage()
+                    }}
+                  >
+                    {contracts
+                      .filter((contract) => !contract.isTerminated)
+                      .map((contract) => (
+                        <MuiMenuItem
+                          key={`${contract.contractTypeName} (${contract.currentAgreementId}`}
+                          value={contract.id}
+                        >
+                          {contract.contractTypeName} (
+                          {contract.currentAgreementId})
```

Comment:
An ID says nothing to IEX, the address on the other hand does
```suggestion
                          {contract.contractTypeName} (
                          {contract.currentAgreement?.address})
```